data/campaigns/atl02.wmf/scripting/texts.lua

```diff
@@ -0,0 +1,743 @@
+   number = 1,
+   body = objective_text(_("Find Portspace"),
+   p(_([[To set foot on this island, we must first find a building space where a port can be built. Explore the nearby coast line to find such a building plot.]])) ..
+   p(_([[After the port is built we should cancel the other ship’s expedition, to unload its wares as well.]])))
+}
+
+obj_basic_infrastructure = {
+   name = "obj_basic_infrastructure",
+   title = _("Ensure the supply of build wares"),
+   number = 4,
+   body = objective_text(_("Establish Basic Production Chain"),
+   p(_([[We will need at least a quarry, a few woodcutters, and a sawmill. A forester might or might not be needed depending on how fast the trees are growing around us. Find the balance.]])))
+}
+
+obj_tools = {
+   name = "obj_tools",
+   title = _("Build tools quickly"),
+   number = 1,
+   body = objective_text(_("Make New Tools"),
+      p(_([[Most of our tools have been lost in the ship tragedy. Our people will not be able to fulfill their duties without proper tools. Luckily we still have some iron ore and plenty of coal. Build a production of tools, as it is a prerequisite for success on the island. Don’t forget to build a smelting works as well.]])))
+}
+
+obj_mining = {
+   name = "obj_mining",
+   title = _("Search the mountains for resources"),
+   number = 3,
+   body = objective_text(_("Build Mines"),
+      p(_([[As we are able to make tools again, our supply of mining resources needs to be replenished soon. Explore the mountains and build at least one mine of each type to mine coal, iron and gold.]])))
+}
+
+obj_explore = {
+   name = "obj_explore",
+   title = _("Explore the surroundings"),
+   number = 1,
+   body = objective_text(_("Explore Your Surroundings"),
+   p(_([[We are fairly sure that we are not alone on the island. We do not know if the others on this island are friends or foes, so we need to tread carefully. Explore the island by building at least one habitat for a scout.]])))
+}
+
+obj_expand = {
+   name = "obj_expand",
+   title = _("Expand your teritory"),
+   number = 1,
+   body = objective_text(_("Expand Territory"),
+   p(_([[As our settlement is going to grow up, we will need more space for all buildings. Expand our territory by building military buildings. Build at least one tower for its larger view range.]])))
+}
+
+obj_spidercloth = {
+   name = "obj_spidercloth",
+   title = _("Build a spider farm and a weaving mill"),
+   number = 2,
+   body = objective_text(_("Spidercloth Production"),
+      p(_([[We need some spidercloth urgently as we are running out of it. It is therefore essential to spare every piece we still have. Reduce the input of all buildings that consume spidercloth, not forgetting the construction sites.]])) ..
+      p(_([[Afterwards we need to construct a farm under all circumstances, as it needs spidercloth to be built. Additionally, we need a spiderfarm and a weaving mill. Later on, a gold spinning mill would be helpful as well.]])))
```

Comment:
Since the gold spinning mill has nothing to do with spidercloth I'd not mention it here